src/schema.ts

```diff
@@ -312,3 +288,17 @@ export interface FieldSchema {
   timeStats?: {[timeUnit: string]: Summary};
   title?: string;
 }
+
+interface Bin {
```

Comment:
This is what datalib returns?  If so, I think this should be in datalib.d.ts.   Also add JsDocs style /** */ comment that this is return from which method.